CHANGES/344.misc

```diff
@@ -0,0 +1,8 @@
+Add bump2version config to setup.cfg
+
+Use bump2version to increment the 'version' string wherever
+it is needed. The canonical source of truth for the version
+is 'bump2version.current_version' in setup.cfg.
+
+To update version, change that and run 'bump2version'.
+
```

Comment:
`towncrier` supports more than a single line in a changelog file, but I'm not sure we need to provide more than 1 line in most cases

For this particular issue, since changelog is focused more towards users than devs, probably don't need extra info

That said, it is moot overall, `.misc` is considered miscellaneous and `towncrier` does not include text in changelog just a link